src/controllers/notificationController.js

```diff
@@ -21,4 +21,20 @@ export default class notificationController {
       return Response.errorResponse(res, 500, res.__('server error'));
     }
   }
+
+  /**
+   * @param  {object} req
+   * @param  {object} res
+   * @param  {object} next
+   * @return {object} notification
+   */
+  static async optOutEmailNotifications(req, res) {
+    try {
+      const { user } = req;
+      await db.User.update({ emailNotifications: false }, { where: { id: user.id } });
+      return Response.success(res, 200, res.__('all unread notifications marked as read'));
+    } catch (error) {
+      return Response.errorResponse(res, 500, res.__('server error'));
```

Comment:
wouldn't it be better if the user is able to turn emailNotications to false or true? so that when the user wants to receive an email notification the user is able to update the email notification to true